page.codeberg.maerchenfeeimgarten.Pfeilspiel.yaml

```diff
@@ -0,0 +1,30 @@
+id: page.codeberg.maerchenfeeimgarten.Pfeilspiel
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: "Pfeilspiel_TNG_international"
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+modules:
+  - name: Pfeilspiel_TNG_international
+    buildsystem: simple
+    build-commands:
+      - install -D  Pfeilspiel_TNG_international /app/bin/Pfeilspiel_TNG_international
+      - install -Dm 0644 page.codeberg.maerchenfeeimgarten.Pfeilspiel.desktop /app/share/applications//page.codeberg.maerchenfeeimgarten.Pfeilspiel.desktop
+      - install -Dm 0644 Pfeilspiel_TNG.svg /app/share/icons/hicolor/scalable/apps/page.codeberg.maerchenfeeimgarten.Pfeilspiel.svg
+    sources:
+      - type: file
+        sha256: 79c40f98189e4ffcf91f0390218e04aedd744be767d833ec5c7aabb64e7586c2
+        only-arches: [x86_64]
+        url: https://codeberg.org/MaerchenfeeimGarten/Pfeilspiel/releases/download/v0.0.8/Pfeilspiel_TNG_international
+      - type: file
+        url: https://codeberg.org/MaerchenfeeimGarten/Pfeilspiel/raw/commit/8710d52eb5d411110432cb3c6d3a4037eac00a33/src/pfeiltng/page.codeberg.maerchenfeeimgarten.Pfeilspiel.desktop
+        sha256: c82c80cea1fdc2d65473370db66fdddf513d0534022c282cc586b21549437a73
+      - type: file
+        url: https://codeberg.org/MaerchenfeeimGarten/Pfeilspiel/raw/commit/8710d52eb5d411110432cb3c6d3a4037eac00a33/src/pfeiltng/Pfeilspiel_TNG.svg
+        sha256: 86348436440c7e1d61351d6c43db828301fe75c4fb94e3cfc68098695fcc0949
+      - type: file
+        url: https://codeberg.org/MaerchenfeeimGarten/Pfeilspiel/raw/commit/95ea1f98ff52a911b17cb2098ee785510954a44f/src/pfeiltng/page.codeberg.maerchenfeeimgarten.Pfeilspiel.metainfo.xml
```

Comment:
as I said previously, this is never installed.

Reply:
Ok, now I have figured it out.